frontend/src/components/indices/IndicesValidate.js

```diff
@@ -0,0 +1,295 @@
+    }
+
+    return newValues
+  }
+
+  return (
+    <>
+      <AppPageHeader
+        title={title}
+        onBack={() => history.push('/indices/list')}
+      />
+      <PageContent>
+      { validationResult
+        ?
+          <div style={{marginBottom: '3rem'}}>
+            <IndicesValidationResult validationResult={validationResult}/>
+            <Button
+              type="primary"
+              htmlType="submit"
+              style={{
+                float:'right',
+                margin: '1rem 5rem 0rem 0rem',
+              }}
+              icon={<RedoOutlined />}
+              onClick={() => setValidationResult()}
+            >
+              Modify validation parameters
+            </Button>
+          </div>
+        :
+          <Form
+            key={'validation'}
+            labelCol={{ span: 4 }}
+            wrapperCol={{ span: 14 }}
+            layout="horizontal"
+            initialValues={formData}
+            onValuesChange={onValuesChange}
+            onFinish={onSubmit}
+            style={{width: '70%'}}
+          >
+            <Form.Item label="Instrument Type" {...props("instrument_type")} rules={requiredRules}>
+              <Select
+                placeholder="Select an instrument type"
+                showSearch
+                allowClear
+                options={instrumentTypes}
+                filterOption={(input, option) =>
+                  option.label.props.children.toLowerCase().indexOf(input.toLowerCase()) >= 0
+                }
+              />
+            </Form.Item>
+            <Form.Item
+              label="Index 3' length"
+              {...props("length_3prime")}
+              extra="Desired sequence length for the index 3 prime"
+            >
+              <InputNumber step={1} />
+            </Form.Item>
+            <Form.Item
+              label="Index 5' length"
+              {...props("length_5prime")}
+              extra="Desired sequence length for the index 5 prime"
+            >
+              <InputNumber step={1} />
+            </Form.Item>
+            <Form.Item
+              label="Indices"
+              hasFeedback
+              validateStatus="warning"
+              {...props("indices")}
+              extra="Indices chosen for validation. Loading some sets may take some time."
+              rules={requiredRules}
+            >
+              <Cascader
+                style={{ width: "100%" }}
+                options={indicesBySet}
+                multiple
+                showSearch={(inputValue, path) =>
+                  path.some(option =>
+                    option.label.toLowerCase().indexOf(inputValue.toLowerCase()) > -1)
+                }
+                loadData={loadData}
+                changeOnSelect
+              />
+            </Form.Item>
+            <Form.Item
+              label="Threshold"
+              {...props("threshold")}
+              extra="Number of allowed errors."
```

Comment:
"...allowed read errors at sequencing time." might be more informative. Lab tech might not be too sure what this is.